enh(release): Prepare release notes for modules 22.10 (#2472)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md

## Centreon MAP

### 22.10.7

Release date: `July 20, 2023`

#### Enhancements

- [UI] You can now hide the label of a resource.
- [Viewer] Resources now have a unique status.
- [Viewer] You can now define a label for a URL shape.

#### Bug fixes

> - [Editor] Fixed a bug where empty metric names were blocking edition.

Reviewer: ```suggestion
- [Editor] Fixed a bug where empty metric min or max prevented users from saving a map.
```

> - [Editor] Fixed metric links when no metric name is filled in.

Reviewer: ```suggestion
- [Editor] Fixed metric links when no metric name were filled in.
```

> - [Server] Fixed an issue which made diagnostic to raise an error if a custom URI is used.

Reviewer: ```suggestion
- [Server] Fixed an issue which made diagnostic.sh raise an error if a custom URI was used.
```

- [UI] Fixed image edition for a MAP thumbnail or media component.
- [UI] Fixed the selection of layers in Geo views.
- [Viewer] Fixed a regression causing output not to be displayed in widget output.

> - [Viewer] Fixed an issue making links not to be reactivated when the associated resource goes back on line.

Reviewer: ```suggestion
- [Viewer] Fixed an issue that caused links not to be reactivated when the associated resource went back online.
```

> - [Viewer] Fixed an issue preventing hosts from being displayed on Geo views.

Reviewer: ```suggestion
- [Viewer] Fixed an issue preventing hosts from being displayed in Geo views.
```

## Centreon BAM

### 22.10.4

Release date: `soon`

#### Bug fixes

> - [UI] Fixed the column name display in the acknowledgment description pop-up.

Reviewer: ```suggestion
- [UI] Fixed the way column names were displayed in the acknowledgment description pop-up.
```

## Centreon MBI

### 22.10.3

Release date: `soon`

#### Bug fixes

- [Compatibility] Fixed the compatibility between Gorgone and MBI by adding the missing no-purge option in eventStatisticsBuilder and perfdataStatisticsBuilder scripts.

> - [Configuration] Removed the image beside "Report specific parameters" that was not well displayed.

Reviewer: ```suggestion
- [Configuration] Removed the image beside "Report specific parameters" that was not displayed properly.
```

- [Install] Added missing Gorgone dependencies required by MBI ETL.

> - [Reporting] Fixed the display of ongoing alert duration in Hostgroups-Service-Current-Events report.

Reviewer: ```suggestion
- [Reporting] Fixed the display of the duration of ongoing alerts in the Hostgroups-Service-Current-Events report.
```
